### ISSUE TITLE
Provide sample deployment code for Azure

### ISSUE THREAD

**Issue by PacoVK** [PacoVK/tapir#47]
Add Terraform example code to deploy Tapir on Azure

**Comment by TomBeckett** [comment]
@PacoVK I've got a TF example that deploys to [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps). Would this be useful?

**Comment by TomBeckett** [comment]
@cedricbraekevelt Absolutely, its fairly rough and ready but I'll raise a PR this weekend 👍

**Comment by PacoVK** [comment]
That would be awesome! Even a rough PR would help as we can iterate on that 🎉

**Comment by TomBeckett** [comment]
@cedricbraekevelt @PacoVK I've raised a PR with an example. 

I've also included a basic `README.md` with some info if you've never deployed Tapir before. Feel free to remove this if you think this is not necessary.
